WordPressCom-Stats-iOS/InsightsTableViewController.m

```diff
@@ -445,4 +674,33 @@ - (UIImage *)commentsImage
     return commentsImage;
 }
 
+#pragma mark - Row and section calculation methods
+
+- (StatsSection)statsSectionForTableViewSection:(NSInteger)section
+{
+    return (StatsSection)[self.sections[(NSUInteger)section] integerValue];
+}
+
+
+- (id)statsDataForStatsSection:(StatsSection)statsSection
+{
+    id data = self.sectionData[@(statsSection)];
+    
+    return data;
```

Comment:
I copied that from the main stats table - removing the method as its not even used.